renew.py

```diff
@@ -179,12 +180,12 @@ def get_cert_data():
         for crt in sorted(glob.glob('/etc/ssl/mysociety/certs/*.crt')):
             with open(crt) as f:
                 cert = OpenSSL.crypto.load_certificate(OpenSSL.crypto.FILETYPE_PEM, f.read())
-            expiry = cert.get_notAfter()
+            expiry = cert.get_notAfter().decode('utf-8')
```

Comment:
`ascii`, maybe? given this should always be an ascii string and presumably you'd want it to error if it somehow got unicode in it :)